src/libcore/num/mod.rs

```diff
@@ -329,6 +329,27 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Returns the number of leading ones in the binary representation of `self`.
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "let n = 0b10110011", stringify!($SelfT), ";
+
+assert_eq!(n.leading_ones(), 1);",
+$EndFeature, "
+```"),
+            #[unstable(feature = "leading_ones_trailing_ones", issue = "0")]
+            #[rustc_const_unstable(feature = "const_int_ops")]
```

Comment:
I'd just remove this; I think it's fairly certain that we'll want it to be `const fn` when stabilizing.